keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone_presets.py

```diff
@@ -0,0 +1,172 @@
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+        },
+    },
+    "efficientnetv2-b0": {
+        "metadata": {
+            "description": ("EfficientNetv2 model "),
+        },
+        "class_name": "keras_cv.models>EfficientNetV2Backbone",
+        "config": {
+            "width_coefficient": 1.0,
+            "depth_coefficient": 1.0,
+            "default_size": 224,
+            "dropout_rate": 0.2,
+            "drop_connect_rate": 0.2,
+            "depth_divisor": 8,
+            "min_depth": 8,
+            "bn_momentum": 0.9,
+            "activation": "swish",
+            "blocks_args": "default",
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+        },
+    },
+    "efficientnetv2-b1": {
+        "metadata": {
+            "description": ("EfficientNetv2 model "),
+        },
+        "class_name": "keras_cv.models>EfficientNetV2Backbone",
+        "config": {
+            "width_coefficient": 1.0,
+            "depth_coefficient": 1.1,
+            "default_size": 240,
+            "dropout_rate": 0.2,
+            "drop_connect_rate": 0.2,
+            "depth_divisor": 8,
+            "min_depth": 8,
+            "bn_momentum": 0.9,
+            "activation": "swish",
+            "blocks_args": "default",
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+        },
+    },
+    "efficientnetv2-b2": {
+        "metadata": {
+            "description": ("EfficientNetv2 model "),
+        },
+        "class_name": "keras_cv.models>EfficientNetV2Backbone",
+        "config": {
+            "width_coefficient": 1.1,
+            "depth_coefficient": 1.2,
+            "default_size": 260,
+            "dropout_rate": 0.2,
+            "drop_connect_rate": 0.2,
+            "depth_divisor": 8,
+            "min_depth": 8,
+            "bn_momentum": 0.9,
+            "activation": "swish",
+            "blocks_args": "default",
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+        },
+    },
+    "efficientnetv2-b3": {
+        "metadata": {
+            "description": ("EfficientNetv2 model "),
+        },
+        "class_name": "keras_cv.models>EfficientNetV2Backbone",
+        "config": {
+            "width_coefficient": 1.2,
+            "depth_coefficient": 1.4,
+            "default_size": 300,
+            "dropout_rate": 0.2,
+            "drop_connect_rate": 0.2,
+            "depth_divisor": 8,
+            "min_depth": 8,
+            "bn_momentum": 0.9,
+            "activation": "swish",
+            "blocks_args": "default",
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+        },
+    },
+}
+
+backbone_presets_with_weights = {}
```

Comment:
Please add presets for all of the efficientnet weights!

https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/weights.py#L81

Reply:
https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/weights.py#L163